bundling/netlify.toml

```diff
@@ -0,0 +1,12 @@
+[[headers]]
+for = "/*"
+[headers.values]
+Access-Control-Allow-Origin = "*"
+
+[[redirects]]
+force = true # ensure that we always redirect
+from = "/" 
```

Comment:
I came from https://github.com/nwtgck/actions-netlify/issues/689. 

Could you try `from = "/*"` instead?

<img width="584" alt="image" src="https://user-images.githubusercontent.com/10933561/143951041-051fe32e-55d1-4943-8718-faa601d0d0c9.png">
https://docs.netlify.com/configure-builds/file-based-configuration/

Reply:
See 5b58b8897979249e937b994f641a15c52ddf3954. Same issue.

Reply:
Now that I think about it, that's actually not the desired configuration. I would like all assets to be served from the URLs that I upload them to.

The redirect was simply added because if anyone visits https://get.grammy.dev with a browser (instead of loading the bundled assets from code), they should be taken to https://grammy.dev.

If there's a chance that the redirects can cause any problems, then we can just drop them for now. This is what I did in fc29e32424252516adefd859856bcac653546ca8 but the problem persists.